.github/workflows/ci.yml

```diff
@@ -79,3 +79,42 @@ jobs:
       - name: ESLint check
         run: pnpm lint
         working-directory: website
+
+  package_to-json-schema_prettier:
+    name: Run Prettier in packages/to-json-schema
+    needs: install
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup pnpm
+        uses: ./.github/actions/pnpm
+      - name: Prettier check
+        run: pnpm format.check
+        working-directory: packages/to-json-schema
+
+  package_to-json-schema_eslint:
+    name: Run ESLint in packages/to-json-schema
+    needs: install
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup pnpm
+        uses: ./.github/actions/pnpm
+      - name: ESLint check
+        run: pnpm lint
+        working-directory: packages/to-json-schema
+
+  package_to-json-schema_vitest:
+    name: Run Vitest in packages/to-json-schema
+    needs: install
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup pnpm
+        uses: ./.github/actions/pnpm
+      - name: Vitest tests
+        run: pnpm test
+        working-directory: packages/to-json-schema
```

Comment:
Could be done cleaner with GH job matrix 🤔

Reply:
I don't know what GH job matrix is but its probably an improvement for another PR